avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -1192,6 +1243,7 @@ def computeGradC(cfg, particles, headerNeighbourGrid, headerNormalGrid,
   cdef double dwdrr, area
   cdef double pikl, flux
   cdef double hk, hl, ck, cl, lambdakl
+  cdef int SPHoption = sphOption
```

Comment:
add sphOption as int sphOption in the def and you dont need this line

Reply:
done